src/parallel/parallel_elem.C

```diff
@@ -145,12 +145,12 @@ Packing<const Elem *>::packable_size (const Elem * const & elem,
   unsigned int total_packed_bcs = 0;
   const unsigned short n_sides = elem->n_sides();
 
-  if (elem->level() == 0)
+  //if (elem->level() == 0)
```

Comment:
I think even at this "hack to see if it's working" stage you might need to still test level() to decide whether to skip edge and shellface BCs, or you'll break the first time you hit a test that has such a BC on an element with children.